.stylelintrc

```diff
@@ -6,6 +6,7 @@
   ],
   "rules": {
     "no-duplicate-selectors": null,
-    "no-descending-specificity": null
+    "no-descending-specificity": null,
+    "property-blacklist": [["font-family"], { "message": "Please use the typography functions from pasteup/typography"}]
```

Comment:
How about adding `font-size` and `line-height` too?

Reply:
Yeah, think that's my plan, `line-height` is slightly more difficult due to usage in vertical centring. But future PR's, doing a rule at a time 👍